demos/tests/cases.py

```diff
@@ -329,13 +330,77 @@ def single_option_cases(key, *args):
         ],
     )),
 
+    PythonDemo(subdirectory='asl_recognition_demo', device_keys=['-d'], test_cases=combine_cases(
+        TestCase(options={'--no_show': None, '-c': DemoFileArg('classes.json'),
+                          '-i': ms_asl_arg('_nz_sivss20'),
+                          '-m_a': ModelArg('asl-recognition-0004'),
+                          '-m_d': ModelArg('person-detection-asl-0001')})
+    )),
+
+    PythonDemo(subdirectory='bert_question_answering_demo', device_keys=['-d'], test_cases=combine_cases(
+        TestCase(options={'-i': "https://en.wikipedia.org/wiki/OpenVINO",
+                          '--questions': ["Which framework does it support","Who is developers"]}),
```

Comment:
```suggestion
                          '--questions': ["What frameworks does OpenVINO support", "Who are developers"]}),
```
Why double quotes?

Reply:
Fixed

Reply:
There are some other strings with double quotes in this file introduced.

Reply:
FWIW, I find that double quotes are generally a better fit for English text, because it tends to include apostrophes.

Reply:
All fixed, @Wovchena  would you resolve it?